webpack/assets/javascripts/react_app/components/FactCharts/FactChartConstants.js

```diff
@@ -0,0 +1,5 @@
+export const FACT_CHART_DATA_REQUEST = 'FACT_CHART_DATA_REQUEST';
```

Comment:
Could we rename to a const without the `DATA`:
- `FACT_CHART_REQUEST`
- `FACT_CHART_SUCCESS`
- `FACT_CHART_FAILURE`

Reply:
Changed